Branch-SDK/src/io/branch/referral/PrefHelper.java

```diff
@@ -1024,6 +1032,26 @@ public void disableSmartSession() {
         BNC_Smart_Session = false;
     }
 
+    public void setRequestMetadata(@NonNull String key, @NonNull String value) {
```

Comment:
unfortunately, these annotations cant account for run time null values, so we remove the entry below.